packages/venia-ui/lib/components/App/__tests__/app.spec.js

```diff
@@ -160,7 +159,8 @@ test('renders a full page with onlineIndicator and routes', () => {
     };
     const { root } = createTestInstance(<App {...appProps} />);
 
-    getAndConfirmProps(root, Navigation);
+    // TODO: Figure out how to mock the React.lazy call to export the component
```

Comment:
Hey dude, is this still valid given you have added a react mock in `venia-ui` mocks?

Reply:
Yes. There is no `Navigation` component, just the stringified version of the React.lazy callback.